lib/src/components/NxCombobox/NxCombobox.tsx

```diff
@@ -0,0 +1,302 @@
+      inputRef.current?.querySelector('input')?.setSelectionRange(prevSearchText.length, value.length);
+      focusFirst();
+    }
+  }, [prevSearchText]);
+
+  function handleOnClick({displayName}: DataItem<T, string>) {
+    onChange(displayName);
+    focusTextInput();
+    setFocusableBtnIndex(null);
+    setShowDropdown(false);
+  }
+
+  return (
+    /*eslint-disable-next-line jsx-a11y/no-static-element-interactions*/
+    <div ref={ref}
+         className={className}
+         onFocus={handleComponentFocus}
+         onBlur={handleComponentBlur}
+         { ...attrs }>
+      <NxTextInput role="combobox"
+                   ref={inputRef}
+                   id={inputId}
+                   validationErrors={validationErrors}
+                   validatable={validatable}
+                   isPristine={!!isPristine}
+                   className='nx-combobox__input'
+                   value={value}
+                   onChange={handleOnChange}
+                   disabled={disabled || undefined}
+                   onKeyDown={handleKeyDown}
+                   aria-autocomplete={autoComplete ? 'both' : 'list'}
+                   aria-expanded={showDropdown && !isAlert}
+                   aria-controls={dropdownId}
+                   aria-activedescendant={focusableBtnIndex !== null ?
+                     dropdownRef.current?.children[focusableBtnIndex].id : undefined }
+                   aria-required={ariaRequired}
+                   aria-describedby={ isAlert ? alertDropdownId : ariaDescribedBy}
+                   aria-label={ariaLabel}/>
+      { isAlert ?
+        <div id={alertDropdownId}
+             role="alert"
+             aria-busy={!!loading}
+             aria-live="polite"
+             className={alertClassName}
+             aria-hidden={!showDropdown}>
+          {loadError ? <NxLoadError error={loadError} retryHandler={() => doSearch(value)} /> :
+          loading ? <NxLoadingSpinner /> :
+          isEmpty && <div className="nx-combobox__empty-message">{emptyMessage || 'No Results Found'}</div>}
+        </div>
+        :
+        <NxDropdownMenu id={dropdownId}
+                        role='listbox'
+                        ref={dropdownRef}
+                        className='nx-combobox__menu'
+                        onClosing={() => {}}
+                        aria-hidden={!showDropdown}>
+          {
+          matches.length && matches.map((match, i) =>
+            <button id={`${dropdownBtnId}-${i}`}
+                    role="option"
+                    aria-selected={i === focusableBtnIndex }
+                    className= {classnames('nx-dropdown-button',
+                        { 'selected': i === focusableBtnIndex })}
+                    tabIndex={-1}
+                    disabled={disabled || undefined}
+                    key={match.id}
+                    onClick={() => handleOnClick(match)}>
+              {match.displayName}
+            </button>
+          )
```

Comment:
Seems like this should be indented further.  Does eslint disagree?

Reply:
It doesn't complain either.
I adjusted the indentation, updated here c18b4ffa478e64baa031b10867db13fb869c3294.